app/resources/views/pages/dominion/op-center/index.blade.php

```diff
@@ -93,6 +93,61 @@
                     </table>
                 </div>
             </div>
+
+            <div class="box box-primary">
+                <div class="box-header">
+                    <h3 class="box-title">Town Crier</h3>
+                </div>
+                <div class="box-body table-responsive">
+                    <table class="table table-hover" id="dominions-table">
+                        <colgroup>
+                            <col>
+                            <col>
+                            <col width="200">
+                        </colgroup>
+                        <thead>
+                            <tr>
+                                <th>Realm</th>
+                                <th>Target</th>
```

Comment:
Is target really needed here?

Reply:
Not necessary, but I don't see why not since there's plenty of space :)